### ISSUE TITLE
Demo data for F4D Converter

### ISSUE THREAD

**Issue by cheonjeongdae** [Gaia3D/mago3djs#592]
Demo data for F4D Converter.

[demo_f4d.zip](https://github.com/Gaia3D/mago3djs/files/2029661/demo_f4d.zip)

**Comment by AnilGutha** [comment]
Unable to convert ifc to f4d through command prompt
_"C:\F4DConverter>F4DConverter.exe -inputFolder C:\demo_f4d -outputFolder C:\f4d\projectname -log C:\demo_f4d/logTest.txt -indexing y"_
it's saying that invalid arguments

**Comment by shire75** [comment]
The prefix, which indicate a string in input command line is a key arguments, has been changed from '-' to '#'.
So, the command line you wrote should be changed to 'F4DConverter.exe #inputFolder C:\demo_f4d #outputFolder C:\f4d\projectname #log C:\demo_f4d/logTest.txt #indexing y'

**Comment by AnilGutha** [comment]
@shire75, Thank you!!

**Comment by sergiorb** [comment]
Hi everyone. I'm following the workshop slide published in [LinkedIn](https://www.slideshare.net/ShinHayan/mago3d-workshopenglish-in-thailand-201807?ref=http://www.mago3d.com/eng/?p=183) but I find the next problem:

When I run the `F4DConverter.exe #inputFolder ...` command I got the same terminal output (it also seems to take some seconds processing the data) as in the slide but no output files nor log file.

Has someone an idea about what I could do?

Thanks you all in advance!

**Comment by Chep2** [comment]
I installed the latest version of F4DConverter, downloaded demo_f4d.zip, created directories.
But on the command line, I get the error:
`[ERROR][Invalid Arguments] #log and #meshType are MANDATORY when #inputFolder is
 used.`
`[Error]Invalid Arguments.`
![f4d](https://user-images.githubusercontent.com/25004523/116285162-ea269300-a79e-11eb-86fb-daec838b202f.jpg)

help me please

**Comment by shire75** [comment]
Hi @Ched2

You should pass '#meshType [typeValue]' argument to the converter when you want to convert.

#meshType 0  -> for semantic 3d model (cad-based or artificially designed data. most of 3d data are this type)

#meshType 3 -> points cloud data of .las format

Thanks.

**Comment by Chep2** [comment]
@shire75, thanks for the answer!
I run the command:
`F4DConverter.exe #inputFolder C:\demo_f4d #outputFolder C:\f4d\projectname #meshType 0 #log C:\demo_f4d\logTest.txt #indexing y`
But I get the error:
![error](https://user-images.githubusercontent.com/25004523/116362667-5c38bf80-a813-11eb-86c1-2ed30a975f3f.jpg)
The directory is empty after that:
![folder](https://user-images.githubusercontent.com/25004523/116363610-60191180-a814-11eb-87bf-9ca5218bc74a.jpg)

I just intended to use this demo data to test run the system.

**Comment by shire75** [comment]
Hi @Chep2 

Did you run the converter in VM environment or remote machine?

In that situation, Opengl v4.x or later is not activated which is necessary for running the converter successfully.

**Comment by Chep2** [comment]
Yes, you're right - this was a test on Windows 7 x64 in VirtualBox.
On my main computer (Windows 10 x64) another error:
![error2](https://user-images.githubusercontent.com/25004523/116424207-5f9f6b80-a852-11eb-8218-6e67b6e12c07.jpg)
It’s strange. After all, I run CMD as administrator ...

**Comment by Chep2** [comment]
I have installed Windows 7 as my second operating system.
The files have been converted.
![cool](https://user-images.githubusercontent.com/25004523/116451297-57a0f500-a86d-11eb-8c00-e8092235f855.jpg)

`693 models out of 5243 meshes detected.`
Is this conversion successful?
Is that how it should be?
